src/components/custom/Toolkits/toolkits-config.ts

```diff
@@ -783,10 +784,10 @@ const comingSoonTools: Tool[] = [
     name: "Stripe",
     image: "stripe.svg",
     summary: "Process payments and manage subscriptions with your agents.",
-    link: "/toolkits/productivity/stripe",
-    category: "productivity",
+    link: "/toolkits/payments/stripe",
+    category: "payments",
```

Comment:
nit: do we need a redirect? Probably not worth it at this moment, but might there be other pages linking here?